swagger.yaml

```diff
@@ -460,6 +461,63 @@ paths:
         "default":
           description: Invalid request was sent to the server.
 
+  "/users/{userId}/vacation":
+   put:
+    operationId: updateUserVacation
+    summary: Update user vacation days
+    description: Update vacation days for a specific user by year.
+    tags:
+      - Users
+    parameters:
+      - name: userId
+        in: path
+        required: true
+        description: The Keycloak ID of the user to update vacation for
+        schema:
+          type: string
+          format: uuid
+    requestBody:
+      required: true
+      content:
+        application/json:
+          schema:
+            type: object
+            properties:
+              vacationDays:
+                type: object
+                description: Vacation days by year (e.g., "2024", "2025")
```

Comment:
You should describe here that the years are the key for the "additionalProperties" value.
I would also suggest creating a schema to replace additionalProperties and call it something more descriptive e.g. YearlyVacationDays. You can then reference the schema here (as is done with User  inthe 200 response below for example) rather than defining it here.